src/main/java/hudson/plugins/jira/selector/jobs/JobIssueSelector.java

```diff
@@ -0,0 +1,34 @@
+package hudson.plugins.jira.selector.jobs;
```

Comment:
Ok. Makes sense. I though there will be other job based SCMs so it would make sense but i can't find any. So let's apply YAGNI principle here.